fcrepo-event-serialization/src/main/java/org/fcrepo/event/serialization/EventSerializer.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.event.serialization;
+
+import static com.hp.hpl.jena.datatypes.xsd.XSDDatatype.XSDdateTime;
+import static com.hp.hpl.jena.rdf.model.ModelFactory.createDefaultModel;
+import static com.hp.hpl.jena.rdf.model.ResourceFactory.createProperty;
+import static com.hp.hpl.jena.rdf.model.ResourceFactory.createResource;
+import static com.hp.hpl.jena.rdf.model.ResourceFactory.createTypedLiteral;
+import static com.hp.hpl.jena.vocabulary.RDF.type;
+import static com.hp.hpl.jena.vocabulary.DCTerms.identifier;
+import static com.hp.hpl.jena.vocabulary.DCTerms.isPartOf;
+import static org.fcrepo.kernel.api.RdfLexicon.PROV_NAMESPACE;
+import static org.fcrepo.kernel.api.observer.OptionalValues.BASE_URL;
+import static org.fcrepo.kernel.api.observer.OptionalValues.USER_AGENT;
+
+import com.hp.hpl.jena.rdf.model.Model;
+import com.hp.hpl.jena.rdf.model.Resource;
+import org.fcrepo.kernel.api.observer.FedoraEvent;
+
+/**
+ * A basic serialization API for Fedora events
+ * @author acoburn
+ */
+public interface EventSerializer {
+
+    /**
+     * Convert an event to an Rdf Model
+     * @param evt the Fedora event
+     * @return an RDF model representing the event
+     */
+    public static Model toModel(final FedoraEvent evt) {
```

Comment:
I was anticipating that (alternate) classes would be spring-injected. Clearly, one option would be to change the TurtleSerializer class to something like RdfSerializer and parameterize the class construction. I don't actually care that much about how this is organized, though. If you prefer to allow users to parameterize class construction like this, that's fine -- it just requires more knowledge of Jena models on the users' part. I like my approach better because it assumes the user/sysadmin knows that they want Turtle or NTriples or RDF/XML but don't necessarily need to care about how Jena creates those serializations (or what the String values are that correspond to those serializations).